README.md

```diff
@@ -471,45 +459,46 @@ documentation](https://docs.rubocop.org/rubocop/configuration.html#inheriting-co
 * When defining binary operators, name the parameter `other`(`<<` and `[]` are
   exceptions to the rule, since their semantics are different).
 
-* The names of predicate methods (methods that return a boolean value) should
-  end in a question mark (i.e. `Array#empty?`). Methods that don't return a
-  boolean, shouldn't end in a question mark.
+* Name predicate methods with a `?`. Predicate methods are methods that return a
+  boolean value.
+
+* Avoid ending method names with a `?` if they don't return a boolean.
+
+* Avoid prefixing method names with `is_`.
+
+  ~~~ruby
+  # bad
+  def is_empty?
+  end
 
-* Method names should not be prefixed with `is_`. E.g. prefer `empty?` over
-  `is_empty?`.
+  # good
+  def empty?
+  end
+  ~~~
 
-* Avoid magic numbers. Use a constant and give it a useful name.
+* Avoid magic numbers. Use a constant and give it a meaningful name.
 
 * Avoid nomenclature that has (or could be interpreted to have) discriminatory
   origins.
 
 ## Comments
 
-* Good comments focus on the reader of the code, by helping them understand the
-  code. The reader may not have the same understanding, experience and knowledge
-  as you. As a writer, take this into account.
+* When writing comments account for the fact that readers might not have the
+  same context as you do.
 
-* A big problem with comments is that they can get out of sync with the code
-  easily. When refactoring code, refactor the surrounding comments as well.
+* Keep surrounding comments in sync when refactoring code.
```

Comment:
Arguably, refactoring is the one case where the comments might not need to change. 😅 

```suggestion
* Keep comments in sync with code.
```